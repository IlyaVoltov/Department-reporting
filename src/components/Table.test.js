import {render} from "@testing-library/react"
import "@testing-library/jest-dom"
import React from "react"
import developersData from "../pages/Development/API/development.json"
import {TableComponent} from "./Table"
import {MemoryRouter} from "react-router-dom"

describe("Тестирование страницы: План развития сотрудников", () => {
	const Application = () => (
		<MemoryRouter initialEntries={["/development"]}>
			<TableComponent
				haveIconButton={false}
				rows={developersData}
				columnNames={[
					{ru: "Разработчик", en: "developer"},
					{ru: "Направление", en: "position"},
					{ru: "Работает в команде с", en: "workStartDate"},
				]}
			/>
		</MemoryRouter>
	)

	it("Проверка на  рендер приложения", () => {
		const tree = render(<Application />)
		//проверка на количество строк на странице
		expect(tree).toMatchSnapshot()
	})
})
