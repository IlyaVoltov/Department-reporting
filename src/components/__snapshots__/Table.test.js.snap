// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Тестирование страницы: План развития сотрудников Проверка на  рендер приложения 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiTableContainer-root css-denxkp-MuiPaper-root-MuiTableContainer-root"
      >
        <table
          aria-label="collapsible table"
          class="MuiTable-root MuiTable-stickyHeader css-xn82ks-MuiTable-root"
        >
          <thead
            class="MuiTableHead-root css-15wwp11-MuiTableHead-root"
            style="background-color: white;"
          >
            <tr
              class="MuiTableRow-root MuiTableRow-head css-1q1u3t4-MuiTableRow-root"
            >
              <th
                class="MuiTableCell-root MuiTableCell-head MuiTableCell-stickyHeader MuiTableCell-sizeMedium css-5zwa29-MuiTableCell-root"
                colspan="2"
                scope="col"
              >
                Разработчик
              </th>
              <th
                class="MuiTableCell-root MuiTableCell-head MuiTableCell-stickyHeader MuiTableCell-sizeMedium css-5zwa29-MuiTableCell-root"
                colspan="1"
                scope="col"
              >
                Направление
              </th>
              <th
                class="MuiTableCell-root MuiTableCell-head MuiTableCell-stickyHeader MuiTableCell-sizeMedium css-5zwa29-MuiTableCell-root"
                colspan="1"
                scope="col"
              >
                Работает в команде с
              </th>
            </tr>
          </thead>
          <tbody
            class="MuiTableBody-root css-apqrd9-MuiTableBody-root"
          >
            <tr
              aria-label="table-row"
              class="MuiTableRow-root css-1q1u3t4-MuiTableRow-root"
            >
              <th
                class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-aavbm7-MuiTableCell-root"
                scope="row"
              >
                Кулагина Ольга
              </th>
              <th
                class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-aavbm7-MuiTableCell-root"
                scope="row"
              >
                software developer (F-E)
              </th>
              <th
                class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-aavbm7-MuiTableCell-root"
                scope="row"
              >
                2020-01-01
              </th>
            </tr>
            <tr
              aria-label="table-row"
              class="MuiTableRow-root css-1q1u3t4-MuiTableRow-root"
            >
              <th
                class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-aavbm7-MuiTableCell-root"
                scope="row"
              >
                Тяпкина Анастасия
              </th>
              <th
                class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-aavbm7-MuiTableCell-root"
                scope="row"
              >
                software developer (F-E)
              </th>
              <th
                class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-aavbm7-MuiTableCell-root"
                scope="row"
              >
                2020-01-01
              </th>
            </tr>
            <tr
              aria-label="table-row"
              class="MuiTableRow-root css-1q1u3t4-MuiTableRow-root"
            >
              <th
                class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-aavbm7-MuiTableCell-root"
                scope="row"
              >
                Степалин Иван
              </th>
              <th
                class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-aavbm7-MuiTableCell-root"
                scope="row"
              >
                software developer (F-E)
              </th>
              <th
                class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-aavbm7-MuiTableCell-root"
                scope="row"
              >
                2020-01-01
              </th>
            </tr>
            <tr
              aria-label="table-row"
              class="MuiTableRow-root css-1q1u3t4-MuiTableRow-root"
            >
              <th
                class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-aavbm7-MuiTableCell-root"
                scope="row"
              >
                Челышева Кристина
              </th>
              <th
                class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-aavbm7-MuiTableCell-root"
                scope="row"
              >
                UI / UX desigener
              </th>
              <th
                class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-aavbm7-MuiTableCell-root"
                scope="row"
              >
                2020-01-01
              </th>
            </tr>
            <tr
              aria-label="table-row"
              class="MuiTableRow-root css-1q1u3t4-MuiTableRow-root"
            >
              <th
                class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-aavbm7-MuiTableCell-root"
                scope="row"
              >
                Иванов Дмиртрий
              </th>
              <th
                class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-aavbm7-MuiTableCell-root"
                scope="row"
              >
                project manager
              </th>
              <th
                class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-aavbm7-MuiTableCell-root"
                scope="row"
              >
                2020-01-01
              </th>
            </tr>
          </tbody>
          <tfoot
            class="MuiTableFooter-root css-9cgc89-MuiTableFooter-root"
          >
            <tr
              class="MuiTableRow-root MuiTableRow-footer css-1q1u3t4-MuiTableRow-root"
            >
              <td
                class="MuiTableCell-root MuiTableCell-footer MuiTableCell-sizeMedium MuiTablePagination-root css-11lyuwt-MuiTableCell-root-MuiTablePagination-root"
                colspan="4"
              >
                <div
                  class="MuiToolbar-root MuiToolbar-gutters MuiToolbar-regular MuiTablePagination-toolbar css-11bfvty-MuiToolbar-root-MuiTablePagination-toolbar"
                >
                  <div
                    class="MuiTablePagination-spacer css-1psng7p-MuiTablePagination-spacer"
                  />
                  <p
                    class="MuiTablePagination-selectLabel css-pdct74-MuiTablePagination-selectLabel"
                    id="mui-2"
                  >
                    Строк на странице
                  </p>
                  <div
                    class="MuiInputBase-root MuiInputBase-colorPrimary css-16c50h-MuiInputBase-root-MuiTablePagination-select"
                    variant="standard"
                  >
                    <select
                      aria-label="rows per page"
                      class="MuiTablePagination-select MuiNativeSelect-select MuiNativeSelect-standard MuiInputBase-input css-m3l1ig-MuiNativeSelect-select-MuiInputBase-input"
                      id="mui-1"
                    >
                      <option
                        class="MuiTablePagination-menuItem"
                        value="5"
                      >
                        5
                      </option>
                      <option
                        class="MuiTablePagination-menuItem"
                        value="10"
                      >
                        10
                      </option>
                      <option
                        class="MuiTablePagination-menuItem"
                        value="25"
                      >
                        25
                      </option>
                      <option
                        class="MuiTablePagination-menuItem"
                        value="-1"
                      >
                        All
                      </option>
                    </select>
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiTablePagination-selectIcon MuiNativeSelect-icon MuiNativeSelect-iconStandard css-10bey84-MuiSvgIcon-root-MuiNativeSelect-icon"
                      data-testid="ArrowDropDownIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M7 10l5 5 5-5z"
                      />
                    </svg>
                  </div>
                  <p
                    class="MuiTablePagination-displayedRows css-levciy-MuiTablePagination-displayedRows"
                  >
                    1–5 из 5
                  </p>
                  <div
                    class="MuiBox-root css-1zye22"
                  >
                    <button
                      aria-label="first page"
                      class="MuiButtonBase-root Mui-disabled MuiIconButton-root Mui-disabled MuiIconButton-sizeMedium css-78trlr-MuiButtonBase-root-MuiIconButton-root"
                      disabled=""
                      tabindex="-1"
                      type="button"
                    >
                      <svg
                        aria-hidden="true"
                        class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                        data-testid="FirstPageIcon"
                        focusable="false"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M18.41 16.59 13.82 12l4.59-4.59L17 6l-6 6 6 6zM6 6h2v12H6z"
                        />
                      </svg>
                    </button>
                    <button
                      aria-label="previous page"
                      class="MuiButtonBase-root Mui-disabled MuiIconButton-root Mui-disabled MuiIconButton-sizeMedium css-78trlr-MuiButtonBase-root-MuiIconButton-root"
                      disabled=""
                      tabindex="-1"
                      type="button"
                    >
                      <svg
                        aria-hidden="true"
                        class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                        data-testid="KeyboardArrowLeftIcon"
                        focusable="false"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M15.41 16.59 10.83 12l4.58-4.59L14 6l-6 6 6 6 1.41-1.41z"
                        />
                      </svg>
                    </button>
                    <button
                      aria-label="next page"
                      class="MuiButtonBase-root Mui-disabled MuiIconButton-root Mui-disabled MuiIconButton-sizeMedium css-78trlr-MuiButtonBase-root-MuiIconButton-root"
                      disabled=""
                      tabindex="-1"
                      type="button"
                    >
                      <svg
                        aria-hidden="true"
                        class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                        data-testid="KeyboardArrowRightIcon"
                        focusable="false"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M8.59 16.59 13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.41z"
                        />
                      </svg>
                    </button>
                    <button
                      aria-label="last page"
                      class="MuiButtonBase-root Mui-disabled MuiIconButton-root Mui-disabled MuiIconButton-sizeMedium css-78trlr-MuiButtonBase-root-MuiIconButton-root"
                      disabled=""
                      tabindex="-1"
                      type="button"
                    >
                      <svg
                        aria-hidden="true"
                        class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                        data-testid="LastPageIcon"
                        focusable="false"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M5.59 7.41 10.18 12l-4.59 4.59L7 18l6-6-6-6zM16 6h2v12h-2z"
                        />
                      </svg>
                    </button>
                  </div>
                </div>
              </td>
            </tr>
          </tfoot>
        </table>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiTableContainer-root css-denxkp-MuiPaper-root-MuiTableContainer-root"
    >
      <table
        aria-label="collapsible table"
        class="MuiTable-root MuiTable-stickyHeader css-xn82ks-MuiTable-root"
      >
        <thead
          class="MuiTableHead-root css-15wwp11-MuiTableHead-root"
          style="background-color: white;"
        >
          <tr
            class="MuiTableRow-root MuiTableRow-head css-1q1u3t4-MuiTableRow-root"
          >
            <th
              class="MuiTableCell-root MuiTableCell-head MuiTableCell-stickyHeader MuiTableCell-sizeMedium css-5zwa29-MuiTableCell-root"
              colspan="2"
              scope="col"
            >
              Разработчик
            </th>
            <th
              class="MuiTableCell-root MuiTableCell-head MuiTableCell-stickyHeader MuiTableCell-sizeMedium css-5zwa29-MuiTableCell-root"
              colspan="1"
              scope="col"
            >
              Направление
            </th>
            <th
              class="MuiTableCell-root MuiTableCell-head MuiTableCell-stickyHeader MuiTableCell-sizeMedium css-5zwa29-MuiTableCell-root"
              colspan="1"
              scope="col"
            >
              Работает в команде с
            </th>
          </tr>
        </thead>
        <tbody
          class="MuiTableBody-root css-apqrd9-MuiTableBody-root"
        >
          <tr
            aria-label="table-row"
            class="MuiTableRow-root css-1q1u3t4-MuiTableRow-root"
          >
            <th
              class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-aavbm7-MuiTableCell-root"
              scope="row"
            >
              Кулагина Ольга
            </th>
            <th
              class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-aavbm7-MuiTableCell-root"
              scope="row"
            >
              software developer (F-E)
            </th>
            <th
              class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-aavbm7-MuiTableCell-root"
              scope="row"
            >
              2020-01-01
            </th>
          </tr>
          <tr
            aria-label="table-row"
            class="MuiTableRow-root css-1q1u3t4-MuiTableRow-root"
          >
            <th
              class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-aavbm7-MuiTableCell-root"
              scope="row"
            >
              Тяпкина Анастасия
            </th>
            <th
              class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-aavbm7-MuiTableCell-root"
              scope="row"
            >
              software developer (F-E)
            </th>
            <th
              class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-aavbm7-MuiTableCell-root"
              scope="row"
            >
              2020-01-01
            </th>
          </tr>
          <tr
            aria-label="table-row"
            class="MuiTableRow-root css-1q1u3t4-MuiTableRow-root"
          >
            <th
              class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-aavbm7-MuiTableCell-root"
              scope="row"
            >
              Степалин Иван
            </th>
            <th
              class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-aavbm7-MuiTableCell-root"
              scope="row"
            >
              software developer (F-E)
            </th>
            <th
              class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-aavbm7-MuiTableCell-root"
              scope="row"
            >
              2020-01-01
            </th>
          </tr>
          <tr
            aria-label="table-row"
            class="MuiTableRow-root css-1q1u3t4-MuiTableRow-root"
          >
            <th
              class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-aavbm7-MuiTableCell-root"
              scope="row"
            >
              Челышева Кристина
            </th>
            <th
              class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-aavbm7-MuiTableCell-root"
              scope="row"
            >
              UI / UX desigener
            </th>
            <th
              class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-aavbm7-MuiTableCell-root"
              scope="row"
            >
              2020-01-01
            </th>
          </tr>
          <tr
            aria-label="table-row"
            class="MuiTableRow-root css-1q1u3t4-MuiTableRow-root"
          >
            <th
              class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-aavbm7-MuiTableCell-root"
              scope="row"
            >
              Иванов Дмиртрий
            </th>
            <th
              class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-aavbm7-MuiTableCell-root"
              scope="row"
            >
              project manager
            </th>
            <th
              class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-aavbm7-MuiTableCell-root"
              scope="row"
            >
              2020-01-01
            </th>
          </tr>
        </tbody>
        <tfoot
          class="MuiTableFooter-root css-9cgc89-MuiTableFooter-root"
        >
          <tr
            class="MuiTableRow-root MuiTableRow-footer css-1q1u3t4-MuiTableRow-root"
          >
            <td
              class="MuiTableCell-root MuiTableCell-footer MuiTableCell-sizeMedium MuiTablePagination-root css-11lyuwt-MuiTableCell-root-MuiTablePagination-root"
              colspan="4"
            >
              <div
                class="MuiToolbar-root MuiToolbar-gutters MuiToolbar-regular MuiTablePagination-toolbar css-11bfvty-MuiToolbar-root-MuiTablePagination-toolbar"
              >
                <div
                  class="MuiTablePagination-spacer css-1psng7p-MuiTablePagination-spacer"
                />
                <p
                  class="MuiTablePagination-selectLabel css-pdct74-MuiTablePagination-selectLabel"
                  id="mui-2"
                >
                  Строк на странице
                </p>
                <div
                  class="MuiInputBase-root MuiInputBase-colorPrimary css-16c50h-MuiInputBase-root-MuiTablePagination-select"
                  variant="standard"
                >
                  <select
                    aria-label="rows per page"
                    class="MuiTablePagination-select MuiNativeSelect-select MuiNativeSelect-standard MuiInputBase-input css-m3l1ig-MuiNativeSelect-select-MuiInputBase-input"
                    id="mui-1"
                  >
                    <option
                      class="MuiTablePagination-menuItem"
                      value="5"
                    >
                      5
                    </option>
                    <option
                      class="MuiTablePagination-menuItem"
                      value="10"
                    >
                      10
                    </option>
                    <option
                      class="MuiTablePagination-menuItem"
                      value="25"
                    >
                      25
                    </option>
                    <option
                      class="MuiTablePagination-menuItem"
                      value="-1"
                    >
                      All
                    </option>
                  </select>
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiTablePagination-selectIcon MuiNativeSelect-icon MuiNativeSelect-iconStandard css-10bey84-MuiSvgIcon-root-MuiNativeSelect-icon"
                    data-testid="ArrowDropDownIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M7 10l5 5 5-5z"
                    />
                  </svg>
                </div>
                <p
                  class="MuiTablePagination-displayedRows css-levciy-MuiTablePagination-displayedRows"
                >
                  1–5 из 5
                </p>
                <div
                  class="MuiBox-root css-1zye22"
                >
                  <button
                    aria-label="first page"
                    class="MuiButtonBase-root Mui-disabled MuiIconButton-root Mui-disabled MuiIconButton-sizeMedium css-78trlr-MuiButtonBase-root-MuiIconButton-root"
                    disabled=""
                    tabindex="-1"
                    type="button"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                      data-testid="FirstPageIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M18.41 16.59 13.82 12l4.59-4.59L17 6l-6 6 6 6zM6 6h2v12H6z"
                      />
                    </svg>
                  </button>
                  <button
                    aria-label="previous page"
                    class="MuiButtonBase-root Mui-disabled MuiIconButton-root Mui-disabled MuiIconButton-sizeMedium css-78trlr-MuiButtonBase-root-MuiIconButton-root"
                    disabled=""
                    tabindex="-1"
                    type="button"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                      data-testid="KeyboardArrowLeftIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M15.41 16.59 10.83 12l4.58-4.59L14 6l-6 6 6 6 1.41-1.41z"
                      />
                    </svg>
                  </button>
                  <button
                    aria-label="next page"
                    class="MuiButtonBase-root Mui-disabled MuiIconButton-root Mui-disabled MuiIconButton-sizeMedium css-78trlr-MuiButtonBase-root-MuiIconButton-root"
                    disabled=""
                    tabindex="-1"
                    type="button"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                      data-testid="KeyboardArrowRightIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M8.59 16.59 13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.41z"
                      />
                    </svg>
                  </button>
                  <button
                    aria-label="last page"
                    class="MuiButtonBase-root Mui-disabled MuiIconButton-root Mui-disabled MuiIconButton-sizeMedium css-78trlr-MuiButtonBase-root-MuiIconButton-root"
                    disabled=""
                    tabindex="-1"
                    type="button"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                      data-testid="LastPageIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M5.59 7.41 10.18 12l-4.59 4.59L7 18l6-6-6-6zM16 6h2v12h-2z"
                      />
                    </svg>
                  </button>
                </div>
              </div>
            </td>
          </tr>
        </tfoot>
      </table>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
